### PR TITLE
feat(route): add VERSE 文化媒體

### LINKED ISSUE #11432: 希望能加入台灣的新聞媒體 Verse
### Category

New media

### Website URL

https://www.verse.com.tw/verse-express/

### Website description

VERSE是一個全新混種的文化媒體，並期許自己是屬於這個時代的文化媒體。

我們雖然以文化為支點，但會討論公共議題，探索正在轉變的價值理念，挖掘正在湧現的文化與社會趨勢，也會剖析創意產業的商業策略。

### What content should be included?

希望可以納入這個專欄裡面的所有文章
https://www.verse.com.tw/articles/rss

### Additional description

_No response_

### This is not a duplicated rss request

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) and [pull requests](https://github.com/DIYgod/RSSHub/pulls) to ensure this rss proposal has not already been requested

### PR DESCRIPTION
**Pull request by miles170**
## 该 PR 相关 Issue / Involved issue

Close #11432

## 完整路由地址 / Example for the proposed route(s)

```routes
/verse/articles
/verse/articles/verse-news
/verse/articles/figure
/verse/articles/culture
/verse/articles/new-ideas
/verse/articles/local
/verse/articles/eating
/verse/articles/features
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [x] 反爬/频率限制 anti-bot or rate limit?
  - [x] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/verse/articles.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const asyncPool = require('tiny-async-pool');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const category = ctx.params.category;
+    const currentUrl = `https://www.verse.com.tw/articles/${category}/`;
```
Comment (TonyRL) [inline lib/v2/verse/articles.js:9]: You are visiting `https://www.verse.com.tw/articles/undefined/` if `category` is not present.
log: https://github.com/DIYgod/RSSHub/actions/runs/3703115344/jobs/6274175147#step:11:15

File: lib/v2/verse/articles.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    const response = await got(currentUrl);
+    const $ = cheerio.load(response.data);
+    const items = [];
+    for await (const data of asyncPool(5, $('.templateA__list .templateA__listTitle a'), (item) => {
```
Comment (TonyRL) [inline lib/v2/verse/articles.js:13]: I don't hit any rate-limiting with 24 requests being sent simultaneously.

File: lib/v2/verse/radar.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'verse.com.tw': {
+        _name: 'VERSE',
+        www: [
+            {
+                title: '專文',
+                docs: 'https://docs.rsshub.app/new-media.html#verse',
+                source: '/articles/:category?',
+                target: '/articles/:category?',
```
Comment (TonyRL) [inline lib/v2/verse/radar.js:9]:
```suggestion
                target: '/verse/articles/:category?',
```

**Comment by miles170** [comment]
Thanks❤️

**Pull request merged**